packages/shared/src/components/feeds/FeedSettings/components/SmartPrompts.tsx

```diff
@@ -114,26 +99,6 @@ export const SmartPrompts = (): ReactElement => {
           </ConditionalWrapper>
         ))}
       </div>
-      {!isPlus && (
```

Comment:
Why did we remove these?

Reply:
It's in the design
https://www.figma.com/design/nmfWPS7x3kzLUvYMkBx2kW/daily.dev---Dev-Mode?node-id=24630-24749&m=dev

Reply:
Oh I see, because they all under plus now :)